### PR TITLE
Added support to other S3 solutions like minio

### LINKED ISSUE #13: Proposal: Add support to another s3 solutions
## Reason or Problem

Docker image doesn't support to set the --endpoint-url parameter, so it is not possible to use it with a non aws s3 solution.

## Proposal or Solution

Add support to set the --endpoint-url parameter to override command's default URL with the given URL.

### PR DESCRIPTION
**Pull request by juanmancebo**
closes #13

### CODE REVIEW

**Comment by juanmancebo** [comment]
pending to review environment variables from repository settings in order to build the project with success.

File: templates/cronjob.yaml [hunk 1]
```diff
@@ -56,6 +56,10 @@ spec:
                 secretKeyRef:
                   name: {{ .Values.s3.secret.name }}
                   key: {{ .Values.s3.secret.keys.secretAccessKey }}
+{{- if .Values.s3.endpoint.enabled }}
```
Comment (rccrdpccl) [inline templates/cronjob.yaml]: Could we just check whether the endpoint url value is null? Maybe `s3.endpointUrl` or `s3.endpoint`?

**Review by rccrdpccl — CHANGES_REQUESTED** [review]
We can change `backup.image.tag` to `0.3.0` as we'll merge it before this

File: templates/cronjob.yaml (hunk 1 above)
Comment (juanmancebo) [reply templates/cronjob.yaml]: sure. I just changed it.

**Review by rccrdpccl — COMMENTED** [review]
Please bump minor version in Chart.yaml, and we're good to go.

File: Chart.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: "0.2.0"
+appVersion: "0.3.0"
 description: Take mysql backups from any mysql instance to AWS S3. Supports GCP CloudSQL
 name: mysql-backup
 icon: https://avatars0.githubusercontent.com/u/170456
-version: 2.1.4
+version: 2.1.5
```
Comment (rccrdpccl) [inline Chart.yaml]: We should bump the minor version (to `2.2.0`) as the change is not (only) a bugfix, but it's adding functionality.

Comment (juanmancebo) [reply Chart.yaml]: you're right. Done

**Review by rccrdpccl — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**